<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DMM_MainDiag</class>
 <widget class="QDialog" name="DMM_MainDiag">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>786</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>521</width>
     <height>391</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLCDNumber" name="lcdValue">
          <property name="numDigits">
           <number>6</number>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QProgressBar" name="pB_neg">
            <property name="value">
             <number>24</number>
            </property>
            <property name="invertedAppearance">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QProgressBar" name="pB_Pos">
            <property name="value">
             <number>24</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QwtPlot" name="qwt_ValPlot">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTextEdit" name="te_output_dbg">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>450</y>
     <width>361</width>
     <height>101</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>580</y>
     <width>740</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPlainTextEdit" name="te_interface">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="plainText">
       <string>/dev/ttyUSB0</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>183</width>
        <height>28</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="2">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="pb_ID">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_Connect">
        <property name="text">
         <string>Connect</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>597</x>
     <y>40</y>
     <width>187</width>
     <height>377</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_9">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QRadioButton" name="rB_Herz">
            <property name="text">
             <string>Herz</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rB_Farad">
            <property name="text">
             <string>Farad</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rB_Ohm">
            <property name="text">
             <string>Ohm</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rB_Ampere">
            <property name="text">
             <string>Ampere</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rB_Volt">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Volt</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QPushButton" name="pb_Up">
            <property name="text">
             <string>Up</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pb_Auto">
            <property name="text">
             <string>Auto</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pb_Down">
            <property name="text">
             <string>Down</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>18</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QSlider" name="hS_rate">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>20</number>
          </property>
          <property name="singleStep">
           <number>1</number>
          </property>
          <property name="pageStep">
           <number>5</number>
          </property>
          <property name="value">
           <number>2</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBelow</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="lcdRate">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>68</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QRadioButton" name="rb_res_L">
          <property name="text">
           <string>Low</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rb_res_M">
          <property name="text">
           <string>Med</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rb_res_H">
          <property name="text">
           <string>High</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QTextEdit" name="te_output_err">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>450</y>
     <width>361</width>
     <height>101</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pB_neg</sender>
   <signal>valueChanged(int)</signal>
   <receiver>pB_Pos</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>275</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>324</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>clicked(QAbstractButton*)</signal>
   <receiver>DMM_MainDiag</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>767</x>
     <y>663</y>
    </hint>
    <hint type="destinationlabel">
     <x>733</x>
     <y>512</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hS_rate</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdRate</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>651</x>
     <y>311</y>
    </hint>
    <hint type="destinationlabel">
     <x>759</x>
     <y>317</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_Connect</sender>
   <signal>clicked(bool)</signal>
   <receiver>te_interface</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>567</x>
     <y>663</y>
    </hint>
    <hint type="destinationlabel">
     <x>136</x>
     <y>581</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
